gtsfm/scene_optimizer.py

```diff
@@ -410,9 +406,35 @@ def save_visualizations(
     return viz_graph_list
 
 
+def get_gt_gtsfm_data(
```

Comment:
gt_gtsfm sounds a bit weird, how about `get_ground_truth_gtsfm_data()`?

Reply:
I wouldn't exactly call this GT, since the tracks can be noisy. I would call pure GT synthetic correspondences raycasted against mesh, w/ GT poses and GT intrinsics.

How about `get_gtsfm_data_w_est_tracks_and_gt_cameras()`?